README.md

```diff
@@ -103,8 +107,15 @@ for more detail on querying.
 
 ## Optional
 
-An alternative to using `curl` for ES config and querying is
-[kibana](https://www.elastic.co/products/kibana)
+An alternative to using `curl` for ES config, querying and also is visualisation [kibana](https://www.elastic.co/products/kibana)
+Use version [5.3.*](https://www.elastic.co/blog/kibana-5-3-1-released) that works with out ES image and follow the installation steps
+[here](https://www.elastic.co/downloads/kibana#ga-release).
+The default port for ES is `9200` and default index is `profiles`
+
+Here is an example query for the `Discover` tab when you choose the `"name", "alternativeName", "address.addressLines", "address.postcode", "doctors"` for the source:
```

Comment:
The only reason I've added the query is because it's in a different format to the `curl` one. I've only added one as a starting point. I can mention `Dev Tools` working with the `curl` queries though.